src/main/java/codechicken/nei/recipe/GuiRecipe.java

```diff
@@ -100,6 +100,8 @@ public abstract class GuiRecipe<H extends IRecipeHandler> extends GuiContainer i
 
     private int yShift = 0;
 
+    private boolean isHeightHackApplied = false;
```

Comment:
Hmm, could be good to add some JavaDoc here to explain that this variable will be `true` if and only if the height hack applies to the current handler, *and* we're currently within the scope of an active `HeightHack` instance. I was thinking that the latter may not be obvious to someone who is seeing this variable for the first time.

Reply:
Good point! 👍